packages/dashboard/src/web3/context.ts

```diff
@@ -99,14 +110,40 @@ export const Web3Context: Web3ProviderType = {
 }
 
 export function createExternalProvider() {
+    const send = (
+        payload: JsonRpcPayload,
+        callback: (error: Error | null, response?: JsonRpcResponse) => void,
+        overrides?: SendOverrides,
+        options?: RequestOptions,
+    ) => {
+        Services.Ethereum.request(
+            {
+                method: payload.method,
+                params: payload.params,
+            },
+            overrides,
+            options,
+        ).then(
+            (result) => {
+                callback(null, {
+                    jsonrpc: '2.0',
+                    id: payload.id as number,
+                    result,
+                })
+            },
+            (error) => {
+                callback(error)
+            },
+        )
+    }
     return {
         isMetaMask: false,
         isStatus: true,
         host: '',
         path: '',
         request: Services.Ethereum.request,
-        send: Services.Ethereum.requestSend,
-        sendAsync: Services.Ethereum.requestSend,
+        send,
```

Comment:
I want to understand how does this replacement make sense?

Reply:
Actually, `send` and `sendAsync` were both deprecated. But we need here to finer control the payload id. Otherwise, we need to send RPC in legacy NodeJS style.